scripts/menu-script/1-main-menu.sh

```diff
@@ -0,0 +1,260 @@
+#!/usr/bin/env bash
+
```

Comment:
I might suggest naming this file either `0-main-menu.sh` (typically we'll start numbering with 0), or something like `generate-plugin-docs.sh`.